localization/english/event_localization/nickname/wc_nickname_events_l_english.yml

```diff
@@ -0,0 +1,16 @@
+﻿l_english:
+ WCNIC.0001.t:0 "The Oldest of Us"
+ WCNIC.0001.desc.m:0 "A long time has passed since you became the ruler of your tribe, and you have noticed your subjects and the surrounding population have begun to refer to you as “Old Murkeye” due to your long reign and the many years it has been since you were but a tadpole. The most distinguished among your subjects proposed the idea of a ceremony to make this nickname official. Everything has been prepared and all that is left for you to do is to step on the rock and give your raiders a gurgling speech befitting a Murloc of your status."
+ WCNIC.0001.a.m:0 "Mrghllghghllghg!"
+ WCNIC.0001.b.m:0 "Mrglrlhglrlg..."
+ WCNIC.0001.tt.a.m:0 "The murloc raiders cheer and gurgle your nickname."
+ WCNIC.0001.tt.b.m:0 "You turn away from the stone and walk away in refusal, and eventually the Murloc crowd dissipates. The nickname of "the Old Murkeye" slowly fades away in time."
+
+ WCNIC.0001.desc.h:0 "It has been many years since you have ascended to the throne of your country and considerably more since you have come of age. In light of this, your council and the most noble courtiers have proposed a celebration of your long reign, suggesting that you should be known in the sagas as “the Old”. The ceremony for your new title has been prepared and can be held at any time. The population will all come together to dine and praise your name."
```

Comment:
```suggestion
 WCNIC.0001.desc.h:0 "It has been many years since you have ascended to the throne of your country and considerably more since you have come of age. In light of this, your council and the most noble courtiers have proposed a celebration of your long reign, suggesting that you should be known in the sagas as “the Old”. The ceremony for your new title has been prepared and can be held at any time. The population will gather to dine and praise your name.."
```

Reply:
Done in f8acb4c6